lib/rust/ensogl/core/src/debug/stats.rs

```diff
@@ -1,24 +1,11 @@
-//! This module provides utilities for gathering runtime performance statistics of the GUI.
-//!
-//! The module provides a structure which defines the statistics we are interested in ([`Stats`]),
-//! and contains methods for modifying as well as retrieving the current values of the statistics
-//! (often also referred to with the shortcut term "stats"). It also provides methods that need
-//! to be called to ensure that some of the statistics are properly calculated per each frame, and
-//! helper utility types for accumulating and summarizing stats over multiple frames. The intention
-//! behind this module is to aid in detecting and debugging possible performance issues in the GUI.
-//!
-//! Note: some statistics will not be collected (the fields will be present but always zero) when
-//! this crate is compiled without the `statistics` feature flag. This is mediated by the
-//! [`if_compiled_with_stats!`] macro. At the time of writing this doc, the affected stats are:
-//!  - `gpu_memory_usage`
-//!  - `data_upload_size`
+//! This module defines a structure gathering statistics of the running engine. The statistics are
+//! an amazing tool for debugging what is really happening under the hood and understanding the
+//! performance characteristics.
```

Comment:
1. The reverted documentation was way nicer, I'd modify it and leave it.
2. You removed the documentation `Note: some statistics will not be collected (the fields will be present but always zero) when this crate is compiled without the 'statistics' feature flag.` - but I don't see a change in the code regarding this behavior. Does this PR change the behavior?